src/Plugin/MediaEmbedDialog/SoundCloud.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Drupal\stanford_media\Plugin\MediaEmbedDialog;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\media\MediaInterface;
+use Drupal\stanford_media\Plugin\MediaEmbedDialogBase;
+use Drupal\stanford_media\Plugin\MediaEmbedDialogInterface;
+
+/**
+ * Changes embedded file media items.
+ *
+ * @MediaEmbedDialog(
+ *   id = "soundcloud",
+ *   media_type = "audio"
+ * )
+ */
+class SoundCloud extends MediaEmbedDialogBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function isApplicable() {
+    if ($this->entity instanceof MediaInterface && $this->entity->bundle() == 'audio') {
+      $source_field = static::getMediaSourceField($this->entity);
+      $field_value = $this->entity->get($source_field)->getString();
+      return strpos($field_value, 'soundcloud') !== FALSE;
```

Comment:
I suppose it is unlikely but would it be possible that some other audio source has soundclound in the value? Eg: A url to itunes with a song titled soundcloud? Perhaps this should be a bit stronger?

Reply:
Should be better now